### DISCUSSION TITLE
Problem avoid coordinates tDCS optimization

### DISCUSSION THREAD

**Discussion by EFraats** [simnibs/simnibs#429] [8. Simulation workflows] [answered]
I am using the tDCS optimization procedure. However, I’m having some difficulty selecting the correct avoidance region.
When I visualize the results, it appears that the entire brain is being treated as an avoidance region, rather than just my coordinates of interest. Do you have any idea what might be going wrong?

 ```
`target SI only, no avoid
opt = opt_struct('TDCSoptimize');
opt.leadfield_hdf = 'leadfield/ernie_leadfield_EEG10-10_UI_Jurak_2007.hdf5';
opt.name = 'optimization/SI_avoid_5elec_avoid';
 
opt.max_total_current = 2e-3;  %2mA
opt.max_individual_current = 2e-3; %2mA
opt.max_active_electrodes = 5;  
 
opt.target.positions = mni2subject_coords([-50, -27, 52], 'm2m_ernie');
opt.target.intensity = 0.6;   %0.2 minimum
opt.avoid(1).positions = mni2subject_coords([-39, -29, 58], 'm2m_ernie');
% Run optimization
`run_simnibs(opt)``
```

![image](https://github.com/user-attachments/assets/d5ea3af4-30fc-42b2-a907-add75c2df3bb)

**Comment by StevenBm** [comment, marked as answer]
My previous posted response was wrong. I've simulated the problem as well and the issue is the scaling in GMSH. The avoid region gets a penalty of 1000 by default. In your simulation, you are avoiding a single point in the brain. Probably that's why GMSH sets the scale to be maximally at 1. If you click the arrow behind avoid 1 and then options and click on max, it will change to 1000. Because you are avoiding a single point, it will be hard to find though, but by using clipping you can! 

![image](https://github.com/user-attachments/assets/5bad601c-7c0e-4ff2-ad8c-421b587d9339)

**Comment by EFraats** [comment]
Thank you @StevenBm .This was really helpful.
